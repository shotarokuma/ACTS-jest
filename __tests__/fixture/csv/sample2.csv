Column1,Column2,Column3,Column4,Column5,Column6,Column7,Column8,Column9,Column10
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
"sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
","sample cell
"
