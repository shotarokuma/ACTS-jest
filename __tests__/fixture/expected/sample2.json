[
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"},
{"Column1":"sample cell\n","Column2":"sample cell\n","Column3":"sample cell\n","Column4":"sample cell\n","Column5":"sample cell\n","Column6":"sample cell\n","Column7":"sample cell\n","Column8":"sample cell\n","Column9":"sample cell\n","Column10":"sample cell\n"}
]
